modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -172,38 +173,8 @@ func (h *HTTPTransport) NewTransport(caddyCtx caddy.Context) (*http.Transport, e
 		}
 	}
 
-	// Set up the dialer to pull the correct information from the context
-	dialContext := func(ctx context.Context, network, address string) (net.Conn, error) {
-		// the proper dialing information should be embedded into the request's context
-		if dialInfo, ok := GetDialInfo(ctx); ok {
-			network = dialInfo.Network
-			address = dialInfo.Address
-		}
```

Comment:
Losing this code means placeholder dial addresses will no longer work.

Reply:
Hmm... I don't understand how though. Where is the dial address being passed through the replacer? The only place I see it happens is in `fillDialInfo`.

Reply:
Yes. And then the request is updated in `h.reverseProxy` which calls `h.directRequest` with the request and the dial info. That subsequently updates the request with the details from DialInfo.